File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex5.c
#include <stdio.h>

main()
{
    //vars
    float precoAtual, novoPreco;

    printf("\t\t\tDesconto de 10%\n\n");

    //Entrada
    printf("Insira o preco atual do produto: ");
    scanf("%f",&precoAtual);

    //Processamento
    novoPreco = precoAtual * 0.9;

    //Saída
    printf("Novo preco: %.2f", novoPreco);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex5.java
//Importando a classe Scanner
import java.util.Scanner;

public class Ex5{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double precoAtual, novoPreco;
	
	System.out.print("Preço de um produto com descconto de 10%\n\n");
	
	//Entrada
	System.out.print("Insira o preço atual do produto: ");
	precoAtual = ent.nextDouble();
	
	//Processamento
	novoPreco = precoAtual * 0.9;
	
	//Saída	
	System.out.println("Preço com desconto: R$ " + novoPreco);
		
	}
}